src/main/java/uk/ac/cam/cl/dtg/segue/dto/LocalAuthDTO.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2020 Connor Holloway
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.cam.cl.dtg.segue.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The DTO which is used to receive the values required for segue authentication.
+ */
+public class LocalAuthDTO {
+    private String randomPadding;
+    private String email;
+    private String password;
+    private boolean rememberMe;
```

Comment:
If this is a primitive, what happens if I don't provide it? Does it go bang, or does it set it to `false`?
You might find Swagger useful to make requests like this:
http://localhost:8080/isaac-api/api-docs/#/auth/authenticateWithCredentials
(You'll need to set the provider to `SEGUE`, then modify the JSON to not include `rememberMe` etc).

Reply:
It defaults to false. Now replaced with a check that it is specifically set.